test/intl402/DateTimeFormat/prototype/formatRangeToParts/pattern-on-calendar.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google Inc.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-initializedatetimeformat
+description: Checks the DateTimeFormat choose different patterns based
+    on calendar.
```

Comment:
I'm afraid this might break some yaml parsers from different test runners. I'll fix these in a follow up patch